Time-Card/DRV/ptp_ocp.c

```diff
@@ -325,6 +325,11 @@ struct ptp_ocp_signal {
 	bool		running;
 };
 
+struct ptp_ocp_serial_port {
+	int line;
+	int baud;
```

Comment:
What's the point of storing the default baud rate?  This can be changed with stty.

Reply:
Isn't the baud rate fixed on FPGA side? With stty you can change the baud rate of the ttySx device, but it actually works only when the speed matches the speed on the device side, right?

Reply:
Yes, but the GNSS port speeds aren't necessarily fixed.  For example, I have a misbehaving/misconfigured UBX receiver here that is running at 19200.  The speed can be changed by writing a UBX config string.

Since the kernel isn't using the baud values (other than the overly-verbose informational display, which could be removed), there doesn't seem to be much point to this patch.

Reply:
From discussion, let's keep this for now - it can always be cleaned up later.